spring-scheduling/src/main/java/com/baeldung/async/AsyncComponent.java

```diff
@@ -20,9 +20,7 @@ public Future<String> asyncMethodWithReturnType() {
         try {
             Thread.sleep(5000);
             return new AsyncResult<>("hello world !!!!");
-        } catch (final InterruptedException e) {
-
-        }
+        } catch (final InterruptedException ignored) {}
```

Comment:
revert this change as it is not different from the existing one

Reply:
will do this.